res/sections/35-Routing.tex

```diff
@@ -0,0 +1,291 @@
+\chapter{Routing}
+
+Routing determine the ``good''\footnote{The minimum path typically} path
+(sequence of routers) thru network from source to destination.
+Routing protocols use graphs as a mathematical abstraction.
+
+\section{Typologies of routing}
+
+Routing can be \textbf{global}, \textbf{decentralized}, \textbf{static} (where
+the route change slowly over time) and \textbf{dynamic} (where the routes change
+quickly over time).
+
+\subsection{Global routing}
+
+In this kind of routing routers have the complete topology of the network, with
+information about links cost.
+Characteristics:
+\begin{itemize}
+\item ``link state'' algorithm
+\item efficient but not feasible
```

Comment:
not feasible in practice, aggiungerei. Inoltre mancano i punti e vrigola da elenco.